### PR TITLE
SSH server option

### PR DESCRIPTION
**Pull request by xicor22**
## Type of Change
- [x] New feature

## Description
Most winutil users are power users and most probably use ssh to remote into their other machine so I created this PR to add an option to enable ssh server. 

## Testing
- Win 11 

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Comment by MyDrift-user** [comment]
Hey @xicor22, thanks for the PR

You did this change on the same branch as the other one, meaning both additions are in both PRs. Could you devide them by using diffrent branches please?

**Comment by xicor22** [comment]
Yea, i finally understand what i did wrong. Will make the changes soon. I'm new to the code contribution side of github 😅

**Comment by MyDrift-user** [comment]
No issue, thanks in advance :)

**Comment by xicor22** [comment]
@MyDrift-user I removed the #2715 changes from this PR. PLease let me know if I need to change anything else.

**Review by Real-MullaC — APPROVED** [review]

File: functions/private/Invoke-WinUtilSSHServer.ps1 [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    Start-Service -Name sshd
+    Set-Service -Name sshd -StartupType Automatic
+
+    # Sets up the ssh-agent service
+    Start-Service 'ssh-agent'
+    Set-Service -Name 'ssh-agent' -StartupType 'Automatic'
+
+    # Confirm the required services are running
+    $service = Get-Service -Name sshd
+    $service1 = Get-Service -Name 'ssh-agent'
```
Comment (momcilovicluka) [inline functions/private/Invoke-WinUtilSSHServer.ps1:27]: Good work! 👍 May i just suggest changing variable names `service` and  `service1` to more meaningful names like `sshDaemonService` and `sshAgentService` or something similar to better indicate what they represent

**Review by ruxunderscore — APPROVED** [review]
Pretty straight forward. I like this addition. Approved.

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**